src/main/webapp/js/user.js

```diff
@@ -0,0 +1,91 @@
+function autocomplete(input, languages) {
+  let currentFocus;
+  input.addEventListener('input', function(e) {
+    let autocomplete = this.value;
+    let language = this.value;
+    const val = this.value;
+    closeAllLists();
+    if (!val) {
+      return false;
+    }
+    currentFocus = -1;
+    autocomplete = document.createElement('div');
```

Comment:
Instead of generating HTML in JS, consider adding a display: none HTML element in the HTML that represents what a suggestion looks like (including all the ids and classes you need), then using https://developer.mozilla.org/en-US/docs/Web/API/Node/cloneNode to copy it. It'll be easier to maintain and visualize.